### PR TITLE
Changed select field error state to on touched for consistency

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-ToddRoberts this will fix the default html selects. I will have a separate PR for the Sky Select Field component.

File: src/app/public/styles/_forms.scss [hunk 1]
```diff
@@ -27,7 +27,7 @@
 }
 
 input.ng-invalid.ng-touched,
-select.ng-invalid.submitted {
+select.ng-invalid.ng-touched {
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/styles/_forms.scss]: Should we keep the old `.submitted` state as well? Would this change anything for consumers if we remove it?

Comment (Blackbaud-TrevorBurch) [reply src/app/public/styles/_forms.scss]: In looking through examples it does not appear that `submitted` is actually an Angular class. So I'm not sure where that was ever coming from? I think we are ok to remove it.

Comment (Blackbaud-AlexKingman) [reply src/app/public/styles/_forms.scss]: Sounds good.

**Pull request merged**